.github/workflows/main.yaml

```
@@ -0,0 +1,49 @@
+            uses: actions/setup-go@v4
+            with:
+              go-version: '1.24'
+
+          - name: Test
+            run: go test -gcflags="-l" -race -v ./...
+
+     # тестовый билд образа
+    build:
+      runs-on: ubuntu-latest     
+      steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)